lib/utils/prometheus.go

```diff
@@ -57,3 +58,33 @@ func BuildCollector() prometheus.Collector {
 		func() float64 { return 1 },
 	)
 }
+
+// CreateGRPCServerMetrics creates server grpc metrics configuration that is to be registered and used by the caller
+// in an openmetrics unary and/or stream interceptor
+func CreateGRPCServerMetrics(latencyEnabled bool, labels prometheus.Labels) *om.ServerMetrics {
+	serverOpts := []om.ServerMetricsOption{om.WithServerCounterOptions(om.WithConstLabels(labels))}
+	if latencyEnabled {
+		histOpts := []om.HistogramOption{
+			om.WithHistogramBuckets(prometheus.ExponentialBuckets(0.001, 2, 16)),
+			om.WithHistogramConstLabels(labels),
+		}
```

Comment:
Histogram has the same configuration for the client and the server side. Can we extract it to a function?